README.md

```diff
@@ -77,19 +97,14 @@ Analyzing 175 revisions...
 ( 12; 175)              /src
 ```
 
-The ranges on the left indicate
-at what revision folders appeared first and got deleted latest.
-You can see a few things in this very output:
-
- * That a branch called "libbs2b-config-header" got deleted
-   at revision 76.
+The ranges on the left indicate at what revision folders appeared first and got deleted latest.
 
- * In which order bs2b plug-ins appeared over time.
+You can see a few things in this output:
+* That a branch called "libbs2b-config-header" got deleted at revision 76.
+* In which order plug-ins in `plugins` appeared over time.
+* That tag "libbs2b-2.2.1" was deleted at the same revision that a tag folder "libbs2b" appeared.
 
- * That tag "libbs2b-2.2.1" was deleted at the same revision that
-   a tag folder "libbs2b" appeared.
-
-Let's see if that tag was _actually_ moved into that tag subfolder.  Yes it was:
+The last item we can verify to see if it was _actually_ moved into that tag subfolder.
```

Comment:
(If this block did fewer changes of whitespace it would have been a lot easier to spot the difference in content, e.g. which words are dropped or added. Please keep that in mind for the future. I usually advise to not touch whitespace at all and do it in dedicated white-space-only commits, if really necessary.)

Reply:
I find "The last item we can verify to see if" a bit weird but I'm not a native speaker. I get the idea to pull "the last item" first but "verify if" seems to be contradictory mix of "check if" and "verify that". If you can think of another version of that sentence, my vote for a slightly different wording.